.agents/skills/cudf-ci-repro/scripts/docker-repro.sh

```
@@ -0,0 +1,146 @@
+    echo ">>> Checking out PR #${PR_NUMBER}..."
+    gh pr checkout "$PR_NUMBER" --repo rapidsai/cudf
+  fi
+
+  echo ">>> Fetching tags..."
+  git fetch https://github.com/rapidsai/cudf.git --tags 2>/dev/null || true
+
+  # --- Step 2: Prepare local-build sed patch if requested ---
+  if [[ "$LOCAL_BUILD" -eq 1 ]]; then
+    echo ">>> Patching ci/*.sh to use local conda build output (auto-reverted on exit)..."
+    trap 'echo ">>> Reverting ci/*.sh patches..."; git checkout -- ci/*.sh 2>/dev/null || true' EXIT
+    sed -ri '/rapids-download-conda-from-github/ s|^([[:space:]]*[A-Z_]*CHANNEL)=.*|\1=${RAPIDS_CONDA_BLD_OUTPUT_DIR}|' ci/*.sh
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)